x-pack/plugins/enterprise_search/public/applications/app_search/components/sample_engine_creation_cta/sample_engine_creation_cta.test.tsx

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import '../../../__mocks__/enterprise_search_url.mock';
+import { mountWithIntl, setMockActions, setMockValues } from '../../../__mocks__';
+
+import React from 'react';
+
+import { EuiButton } from '@elastic/eui';
+
+import { SampleEngineCreationCta } from './sample_engine_creation_cta';
+
+describe('SampleEngineCTA', () => {
+  describe('CTA button', () => {
+    const MOCK_VALUES = {
+      isLoading: false,
+    };
+
+    const MOCK_ACTIONS = {
+      createSampleEngine: jest.fn(),
+    };
+
+    beforeEach(() => {
+      jest.clearAllMocks();
+      setMockActions(MOCK_ACTIONS);
+      setMockValues(MOCK_VALUES);
+    });
+
+    it('calls createSampleEngine on click', () => {
+      const wrapper = mountWithIntl(<SampleEngineCreationCta />);
```

Comment:
You only need to use `mountWithIntl` if you are using `FormattedMessage`. I believe you could `shallow` in this file.